cmd/updatetree/main.go

```diff
@@ -73,7 +74,12 @@ func updateTree(ctx context.Context) (*trillian.Tree, error) {
 		UpdateMask: treeStateMask,
 	}
 
-	conn, err := grpc.Dial(*adminServerAddr, grpc.WithInsecure())
+	dialOpts, err := rpcflags.NewClientDialOptionsFromFlags()
+	if err != nil {
+		return nil, err
```

Comment:
Maybe decorate the error here like you do in `deletetree/main.go` for consistency?

Reply:
Done.